README.md

```diff
@@ -1,70 +1,147 @@
-# Getting Started with Create React App
+<a name="readme-top"></a>
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+<div align="center">
 
-## Available Scripts
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
 
```

Comment:
- [ ] This image is broken on your readme template. This is probably because you haven't added the [murple_logo.png](https://github.com/microverseinc/readme-template/blob/master/murple_logo.png) image to your project. Kindly add it for it to appear